src/main/java/com/example/home_work_6/Employee.java

```diff
@@ -0,0 +1,68 @@
+package com.example.home_work_6;
+
+public class Employee {
+    private String yorName;
+    private String position;
+    private String eMail;
+    private int tel;
```

Comment:
Would be better to rename this variable to "phoneNumber" for better understanding. "tel" is clear for those who speak russian or ukrainian.